framework/src/source/Coverage.bs

```diff
@@ -5,39 +5,54 @@ namespace rooibos.Coverage
       ? "There was no rooibos code coverage component - not generating coverage report"
       return
     end if
+    t = createObject("roTimespan")
     ? ""
     ? "...Generating code coverage report"
     ? ""
     m.global._rbs_ccn.save = true
     cc = m.global._rbs_ccn
+    expectedMap = cc.expectedMap
+    filePathMap = cc.filePathMap
+    resolvedMap = cc.resolvedMap
```

Comment:
Do these result in 3 distinct copies? If so, could we instead restructure the underlying data type to be a single object instead of 3 separate ones? Would that improve performance at all?

Reply:
Where this only happens after all the tests are done there is no real addition gains to be had here by doing 1 big copy vs 3 individual copies as each one is different data already.